### PR TITLE
Add reference page for 'gsctl list keypairs'

### PR DESCRIPTION
**Pull request by marians**
This adds the reference page for the `gsctl list keypairs` command.

Depends on https://github.com/giantswarm/gsctl/pull/230, as it documents the added `--full` flag.

### CODE REVIEW

File: content/reference/gsctl/list-keypairs.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+# `gsctl list keypairs`
+
+The `gsctl list keypairs` lists information on key pairs for a cluster. This is useful to
+understand which key pairs have been issued when and, if details are given,
+for what purpose. It is also helpful for setting up
+[role based access control](/guides/securing-with-rbac-and-psp/) (RBAC) rules
+in the cluster.
+
+Note that key pair information can only by shown as long as the key pair is
```
Comment (rossf7) [inline content/reference/gsctl/list-keypairs.md]: Marginal but I'd say this should be "has not expired".

File: content/reference/gsctl/list-keypairs.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+not expired.
+
+## Key pairs {#definition}
+
+Giant Swarm offers [X.509](https://en.wikipedia.org/wiki/X.509) client
+certificates as a means to authenticate with the Kubernetes API of a cluster.
+As usual, a certificate comes in pairs with a private key, hence the name
+"key pair".
```
Comment (rossf7) [inline content/reference/gsctl/list-keypairs.md]: The explanation of the name here is nice.

File: content/reference/gsctl/list-keypairs.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+gsctl allows to issue a new key pair using the two commands
```
Comment (rossf7) [inline content/reference/gsctl/list-keypairs.md]: allows you to issue

**Comment by marians** [comment]
Thanks for the reviews!

**Pull request merged**